test/e2e/scheduling/limit_range.go

```diff
@@ -222,6 +226,89 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should ensure that a limitRange can be listed, patched and deleted by collection", func() {
+
+		lrClient := f.ClientSet.CoreV1().LimitRanges(f.Namespace.Name)
+		lrName := "e2e-limitrange-" + utilrand.String(5)
+		createdLabel := map[string]string{lrName: "created"}
+		createdLabelSelector := labels.SelectorFromSet(createdLabel).String()
+		patchedLabelSelector := lrName + "=patched"
+
+		min := getResourceList("50m", "100Mi", "100Gi")
+		max := getResourceList("500m", "500Mi", "500Gi")
+		defaultLimit := getResourceList("500m", "500Mi", "500Gi")
+		defaultRequest := getResourceList("100m", "200Mi", "200Gi")
+		maxLimitRequestRatio := v1.ResourceList{}
+		value := strconv.Itoa(time.Now().Nanosecond()) + string(uuid.NewUUID())
+
+		limitRange := &v1.LimitRange{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: lrName,
+				Labels: map[string]string{
+					"time": value,
```

Comment:
what is this label for?

Reply:
Nothing now, was part of the initial code I was using to understand limitRange. Removing it.